### ISSUE TITLE
Climbing issues

### ISSUE THREAD

**Issue by mazes-80** [runsy/petz#91]
Squirrels are able to climb walls if they are made from any logs.

As a suggestion: 'node.param1' is used to identify placed nodes

you can use:
if node.param1 == 0 then
&nbsp;&nbsp;--do the climb
end

As planks are always placed manually, just disable the ability to climb group:wood things.

Allowing to climb produces weird things, and not allowing also.
So there may be a user setting to allow disallow climbing placed nodes.

**Comment by runsy** [comment]
Yes I know that squirrels go crazy in some situations. I have to redo/adjust their behaviour.
